scripts/process_cams_to_csv.py

```
@@ -0,0 +1,253 @@
+            print(f"  Max: {df[col].max():.1f}")
+            
+            # Check WHO guidelines
+            who_24hr = 15
+            exceed = (df[col] > who_24hr).sum()
+            print(f"\nWHO 24-hr guideline ({who_24hr} µg/m³) exceedances: {exceed}/{len(df)} ({exceed/len(df)*100:.1f}%)")
+            
+        print("="*60 + "\n")
+        
+        return df
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)